addons/website_slides/models/slide_slide.py

```diff
@@ -841,31 +841,18 @@ def _action_vote(self, upvote=True):
         ])
         slide_id = slide_partners.mapped('slide_id')
         new_slides = self_sudo - slide_id
-        channel = slide_id.channel_id
-        karma_to_add = 0
 
         for slide_partner in slide_partners:
             if upvote:
-                new_vote = 0 if slide_partner.vote == 1 else 1
+                slide_partner.vote = 0 if slide_partner.vote == 1 else 1
             else:
-                new_vote = 0 if slide_partner.vote == -1 else -1
-            # 2 if the disliked slide was liked by the user
-            # 1 if the slide was liked OR if the dislike was removed
-            # -1 if the slide was disliked OR if the like was removed
-            # -2 if the liked slide was disliked by the user
-            vote_diff = new_vote - slide_partner.vote
-            karma_to_add += channel.karma_gen_slide_vote * vote_diff
```

Comment:
We removed that feature completely?

Should be explained in the commit message then, or maybe even extracted into a preliminary commit to avoid messing with the refactoring here.